components/gantt/gantt-tree/filter/overview.md

```diff
@@ -21,3 +21,112 @@ To enable filtering, set the grid's `FilterMode` property to one of the followin
 The behavior of the filter input and the available filter operators will depend on the column data type. For example, a `boolean` field will only have the options `"is true"` and `"is false"` and will not have operators like `"contains"` or `"greater than"`.
 
 You can filter more than one column at a time, and all filter rules will be applied together with an `AND` logic.
+
+
+## Customize The Filtering Fields
+
+You can customize the editors rendered in the Gantt
+by providing the `FilterEditorType` attribute, exposed on the `<GanttColumn>`. The `FilterEditorType` attribute accepts a member of the `GanttTreeListFilterEditorType` enum:
+
+| Field data type | GanttTreeListFilterEditorType enum members |
+|-----------------|------------------------------------------|
+| **DateTime**  | `GanttTreeListFilterEditorType.DatePicker`<br> `GanttTreeListFilterEditorType.DateTimePicker` |
+
+
+````CSHTML
+@* The usage of the FilterEditorType parameter *@
+
+<TelerikGantt Data="@Data"
+               Width="100%"
```

Comment:
```suggestion
               Width="900px"
```
If not placed in a container, setting the width to 100% expands the component too much and is not well visible.